editor/editor_run.cpp

```diff
@@ -239,6 +240,9 @@ Error EditorRun::run(const String &p_scene, const String &p_custom_args, const L
 		Error err = OS::get_singleton()->create_process(exec, args, &pid);
 		ERR_FAIL_COND_V(err, err);
 		pids.push_back(pid);
+		if (DebugAdapterProtocol::get_singleton()->is_active()) {
+			DebugAdapterProtocol::get_singleton()->notify_process(pid);
```

Comment:
This probably should listen to `EditorNode` "play_pressed" signal. But do you really need the PID? Because it might not always be available (e.g. when running a non-native export, e.g. debugging HTML5, or android).

Reply:
The PID, `isLocalProcess` and `pointerSize` are all optional, and the last two are difficult/impossible to know on the scenarios you mentioned, so I'll skip them as well.